source/macos/AdaptiveCards/AdaptiveCards/Renderers/AdaptiveCardRenderer.swift

```diff
@@ -59,7 +59,7 @@ class AdaptiveCardRenderer {
         }
         
         // add selectAction
-        rootView.setupSelectAction(selectAction: card.getSelectAction(), rootView: rootView)
+        rootView.target = rootView.setupSelectAction(selectAction: card.getSelectAction(), rootView: rootView)
```

Comment:
Yea, this is not necessary.
`rootView.setupSelectAction(selectAction: card.getSelectAction(), rootView: rootView)` should only be used.
That method should take care of all the configurations.